spec/requests/api/alerts_spec.rb

```diff
@@ -0,0 +1,23 @@
+describe "Alerts API" do
+  let(:alert_status)  { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_status2) { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_url) { alerts_url(alert_status.id) }
+  let(:vm) { FactoryGirl.create(:vm) }
+  let(:alert_definition) { FactoryGirl.create(:miq_alert, :severity => "info") }
+  let(:user) { FactoryGirl.create(:user_with_group) }
+
+  it "forbids access to actions without an appropriate role" do
+    api_basic_authorize
+    run_get(alerts_url)
+    expect(response).to have_http_status(:forbidden)
+  end
+
+  it "reads all alerts statuses" do
+    api_basic_authorize collection_action_identifier(:alerts, :read, :get)
+    alert_status
+    alert_status2
+    run_get(alerts_url)
+    expect(response).to have_http_status(:ok)
+    expect(response.parsed_body["count"]).to eq(2)
+  end
```

Comment:
Preference is that #13323 gets merged in first, then this PR can be completed.  If you cannot wait for that, then update this PR to remove the unused code.